app/models/user.rb

```diff
@@ -45,7 +45,13 @@ class User < ApplicationRecord
   has_many :sign_in_restrictions, dependent: :destroy
   has_many :in_person_enrollments, dependent: :destroy
 
-  has_one :pending_in_person_enrollment, -> { where(status: :pending).order(created_at: :desc) },
+  has_one :pending_in_person_enrollment,
+          -> { where(status: :pending).order(created_at: :desc) },
+          class_name: 'InPersonEnrollment', foreign_key: :user_id, inverse_of: :user,
+          dependent: :destroy
+
+  has_one :establishing_in_person_enrollment,
+          -> { where(status: :establishing, profile: nil).order(created_at: :desc) },
```

Comment:
> May not need a nil profile?

Yep! This turned out to be the issue with the GPO feature spec failure, since we are assigning a profile during the password re-entry, and therefore the GPO verification's attempt to reference the `establishing_in_person_enrollment` resulted in a `nil` value. Removing that from the `where` clears it up.

Reply:
Fixed in b487cadad.